dll/win32/shimgvw/shimgvw.c

```diff
@@ -710,7 +838,7 @@ ImageView_CreateWindow(HWND hwnd, LPWSTR szFileName)
     WndClass.style          = CS_HREDRAW | CS_VREDRAW;
     WndClass.hIcon          = LoadIcon(hInstance, MAKEINTRESOURCE(IDI_APPICON));
     WndClass.hCursor        = LoadCursor(hInstance, IDC_ARROW);
-    WndClass.hbrBackground  = (HBRUSH)COLOR_WINDOW;
+    WndClass.hbrBackground  = (HBRUSH)GetStockObject(NULL_BRUSH);   /* less flicker */
```

Comment:
Assuming you're repainting the entire HDC, setting this to NULL should eliminate the flicker. There's no reason to paint the background and then the foreground in the majority of cases.